tests/test_powerspectrum.py

```diff
@@ -234,9 +234,9 @@ def test_classical_significances_threshold(self):
 
         pval = ps.classical_significances(threshold=threshold,
                                           trial_correction=False)
-
-        assert pval[0][0] < threshold
-        assert pval[0][1] == index
+        pval_list = list(pval)
```

Comment:
Its because in Python 2, the pval would return a list but in 3, its still a zip object and so the length couldn't be taken unless casted to list